base-eslintrc.json

```diff
@@ -0,0 +1,249 @@
+        "allowNumber": true,
+        "allowBoolean": true,
+        "allowAny": false,
+        "allowNullish": true
+      }
+    ],
+    "arrow-parens": ["error", "as-needed"],
+    "comma-dangle": "error",
+    "complexity": "off",
+    "constructor-super": "error",
+    "curly": ["error", "multi-line"],
+    "dot-notation": "off",
+    "eqeqeq": ["error", "smart"],
+    "guard-for-in": "error",
+    "id-denylist": "error",
+    "id-match": "error",
+    "import/order": [
+      "error",
+      {
+        "alphabetize": {
+          "caseInsensitive": true,
+          "order": "asc"
+        },
+        "newlines-between": "ignore",
+        "groups": [
+          ["builtin", "external", "internal", "unknown", "object", "type"],
+          "parent",
+          ["sibling", "index"]
+        ],
+        "distinctGroup": false,
+        "pathGroupsExcludedImportTypes": [],
+        "pathGroups": [
+          {
+            "pattern": "./",
+            "patternOptions": {
+              "nocomment": true,
+              "dot": true
+            },
+            "group": "sibling",
+            "position": "before"
+          },
+          {
+            "pattern": ".",
+            "patternOptions": {
+              "nocomment": true,
+              "dot": true
+            },
+            "group": "sibling",
+            "position": "before"
+          },
+          {
+            "pattern": "..",
+            "patternOptions": {
+              "nocomment": true,
+              "dot": true
+            },
+            "group": "parent",
+            "position": "before"
+          },
+          {
+            "pattern": "../",
+            "patternOptions": {
+              "nocomment": true,
+              "dot": true
+            },
+            "group": "parent",
+            "position": "before"
+          }
+        ]
+      }
+    ],
+    "jsdoc/check-alignment": "error",
+    "jsdoc/check-indentation": "error",
+    "jsdoc/newline-after-description": "off",
+    "max-classes-per-file": "off",
+    "max-len": "off",
+    "new-parens": "error",
+    "no-bitwise": "off",
+    "no-caller": "error",
+    "no-cond-assign": "error",
+    "no-console": "off",
+    "no-debugger": "error",
+    "no-empty": "off",
+    "no-empty-function": "off",
+    "no-eval": "error",
+    "no-fallthrough": "off",
+    "no-invalid-this": "off",
+    "no-new-wrappers": "error",
+    "no-shadow": "off",
+    "no-throw-literal": "error",
+    "no-trailing-spaces": "error",
+    "no-undef-init": "error",
+    "no-underscore-dangle": "off",
+    "no-unsafe-finally": "error",
+    "no-unused-expressions": "off",
+    "no-unused-labels": "error",
+    "no-use-before-define": "off",
+    "no-var": "error",
+    "object-shorthand": "error",
+    "one-var": ["error", "never"],
+    "prefer-arrow/prefer-arrow-functions": ["warn", {}],
+    "prefer-const": "error",
+    "quote-props": ["error", "as-needed"],
+    "quotes": "off",
+    "radix": "error",
+    "semi": "off",
```

Comment:
thought we were about to have a sad convo about semi's, but I see the @typescript-eslint/semi has it covered 🥴